app/requirements.txt

```diff
@@ -4,23 +4,23 @@
 #
 #    pip-compile app/requirements.in
 #
-aiohappyeyeballs==2.4.0
+aiohappyeyeballs==2.4.8
     # via aiohttp
 aiohttp==3.10.5
```

Comment:
Not updated. Should aiohttp be included in requirements.in to be updated?

Reply:
this update will come with the new sdk, you see in the comment in line 10 where this comes from